docs/tutorial-code/1-add-new-technology/generate_models.py

```diff
@@ -0,0 +1,95 @@
+import os
+import shutil
+from pathlib import Path
+
+import pandas as pd
+from muse import examples
+from muse.wizard import (
+    add_new_commodity,
+    add_new_process,
+    add_price_data_for_new_year,
+    modify_toml,
+)
+
+parent_path = Path(__file__).parent
+
+
+"""
+Model 1 - Introduction
+
+"""
+
+
+def generate_model_1() -> None:
```

Comment:
Maybe add a minimal docstring, if just because Alex is working on adding missing docstrings and this function (and similar ones) will be flagged as not up to standards.